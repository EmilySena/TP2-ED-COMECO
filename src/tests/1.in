100
HbvZeMwpyQ 59435 64135
bsCtmvnnIg 85763 480729
hpVzLJpkTH 59435 209154
lPGRdmUBth 85763 690320
pGSuNHGOeT 3581 294259
YQNyOQnrKQ 51579 815852
JaevTirKWf 59435 141325
aLoJYgjPRO 3581 127097
iiBzhZrczr 3581 192337
uuhaJOjytb 59435 705539
rZxQrdLYzv 59435 2777
tRaqxMwDci 85763 253304
VRfZKrQrKk 51579 969357
SFnJwHNTwh 3581 669316
RHquxBafCc 59435 46171
SBmyfhvIIx 85763 659038
spMHlDmGat 51579 715160
sHhtcBKkFk 85763 708349
BlaMjQAwIB 59435 348600
UctZebKIyq 3581 675177
PlXrGUgJCd 3581 487086
QaxzhyBXUZ 3581 733753
UOoICmyajF 3581 333460
qEWTkvQGtY 85763 141334
LWFWFpBBha 85763 191772
CdRWXZkrQo 59435 361104
TarGYkPVTC 3581 664936
JoBZjlzrFn 85763 737624
ebbOYPzDOY 85763 94358
zkCUTTXHvQ 3581 48198
lteoIrEzdJ 51579 955277
rIEGYdwOfo 59435 804859
mriPtSHDXI 3581 810647
VsFSKzuPYL 59435 447369
IkaKJZPieJ 85763 559902
pNmFXyScKr 3581 911307
qRwcoNvcEc 59435 270096
OcwNxUrVIL 85763 984110
AdNjdTUYVE 59435 87392
KTIpBKrRJZ 51579 841161
YYvvBTLUMJ 3581 94854
nczIxwHfRY 85763 389753
cDPuNtMOug 51579 52215
vvWlPykEQc 85763 882397
LwCPnWvAdt 59435 884641
sOjcBkFqDK 3581 977466
FDDPFMBiad 3581 131037
hnhBeKaaaz 3581 628255
fmUPPHIjcw 3581 889505
QJjwgoSblz 51579 802362
brnVPGcNmp 51579 730592
zDMhldpzNs 51579 130401
HmuPdaEQtk 51579 653267
RuZJNeuZEI 85763 819705
LFFAWLiCEm 51579 820226
dJdwxDtUdv 85763 432417
RpWCElzfAx 85763 635607
wjzsngfzSB 85763 495633
FxDQmpAydo 3581 267890
dwpfcRKhib 85763 383872
RtFcyYBSmF 85763 702087
bdflDmLEoH 85763 951208
mkyXWDTpNl 85763 519387
vHLyMoufql 59435 654527
htInLFeiyU 85763 935210
RamQCcGRJJ 51579 11118
SuzvNSiGpD 3581 745116
grlVPIzqKx 59435 478340
bwXkjwpJPu 51579 875874
dJABtlrLkD 85763 998624
jiHxhmplrg 59435 576198
jlsZHcBDgj 3581 550001
vxMsnkFlbZ 3581 557481
maTkRcZlKy 59435 18072
SoCtWesKaj 59435 665930
MjCXenNAjO 3581 953009
XuPMsbmOKU 51579 184537
UWtxLHkTML 59435 543200
TNokAuuwmz 51579 582877
lLxfYyWePg 59435 891487
STtNyjYJUi 51579 357082
jZSXsdSUXR 85763 1308
qZLOWxfWCZ 51579 780834
oKQoAekiyw 51579 775812
wZbJCxqQpn 59435 518870
SABThXOsmY 51579 328669
lmbjpfPifo 51579 969836
JBBGRwPvRW 51579 893459
LldCaoJWow 51579 179595
wPqzzqGBsX 59435 619772
xtmOyAtSgJ 85763 157188
vmjTKEhodV 59435 793535
lkfSNsiPkM 59435 410979
udrWnAfoWh 51579 726306
NexUSvVbTe 59435 256261
XLSKywHgtZ 51579 655635
lobqQIPoHn 3581 881772
QlWRxVPrhp 3581 893780
OSxkuOqmni 51579 544294
FoEyTpKxun 3581 113394
