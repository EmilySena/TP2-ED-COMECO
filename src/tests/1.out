sOjcBkFqDK 3581 977466
MjCXenNAjO 3581 953009
pNmFXyScKr 3581 911307
QlWRxVPrhp 3581 893780
fmUPPHIjcw 3581 889505
lobqQIPoHn 3581 881772
mriPtSHDXI 3581 810647
SuzvNSiGpD 3581 745116
QaxzhyBXUZ 3581 733753
UctZebKIyq 3581 675177
SFnJwHNTwh 3581 669316
TarGYkPVTC 3581 664936
hnhBeKaaaz 3581 628255
vxMsnkFlbZ 3581 557481
jlsZHcBDgj 3581 550001
PlXrGUgJCd 3581 487086
UOoICmyajF 3581 333460
pGSuNHGOeT 3581 294259
FxDQmpAydo 3581 267890
iiBzhZrczr 3581 192337
FDDPFMBiad 3581 131037
aLoJYgjPRO 3581 127097
FoEyTpKxun 3581 113394
YYvvBTLUMJ 3581 94854
zkCUTTXHvQ 3581 48198
lmbjpfPifo 51579 969836
VRfZKrQrKk 51579 969357
lteoIrEzdJ 51579 955277
JBBGRwPvRW 51579 893459
bwXkjwpJPu 51579 875874
KTIpBKrRJZ 51579 841161
LFFAWLiCEm 51579 820226
YQNyOQnrKQ 51579 815852
QJjwgoSblz 51579 802362
qZLOWxfWCZ 51579 780834
oKQoAekiyw 51579 775812
brnVPGcNmp 51579 730592
udrWnAfoWh 51579 726306
spMHlDmGat 51579 715160
XLSKywHgtZ 51579 655635
HmuPdaEQtk 51579 653267
TNokAuuwmz 51579 582877
OSxkuOqmni 51579 544294
STtNyjYJUi 51579 357082
SABThXOsmY 51579 328669
XuPMsbmOKU 51579 184537
LldCaoJWow 51579 179595
zDMhldpzNs 51579 130401
cDPuNtMOug 51579 52215
RamQCcGRJJ 51579 11118
lLxfYyWePg 59435 891487
LwCPnWvAdt 59435 884641
rIEGYdwOfo 59435 804859
vmjTKEhodV 59435 793535
uuhaJOjytb 59435 705539
SoCtWesKaj 59435 665930
vHLyMoufql 59435 654527
wPqzzqGBsX 59435 619772
jiHxhmplrg 59435 576198
UWtxLHkTML 59435 543200
wZbJCxqQpn 59435 518870
grlVPIzqKx 59435 478340
VsFSKzuPYL 59435 447369
lkfSNsiPkM 59435 410979
CdRWXZkrQo 59435 361104
BlaMjQAwIB 59435 348600
qRwcoNvcEc 59435 270096
NexUSvVbTe 59435 256261
hpVzLJpkTH 59435 209154
JaevTirKWf 59435 141325
AdNjdTUYVE 59435 87392
HbvZeMwpyQ 59435 64135
RHquxBafCc 59435 46171
maTkRcZlKy 59435 18072
rZxQrdLYzv 59435 2777
dJABtlrLkD 85763 998624
OcwNxUrVIL 85763 984110
bdflDmLEoH 85763 951208
htInLFeiyU 85763 935210
vvWlPykEQc 85763 882397
RuZJNeuZEI 85763 819705
JoBZjlzrFn 85763 737624
sHhtcBKkFk 85763 708349
RtFcyYBSmF 85763 702087
lPGRdmUBth 85763 690320
SBmyfhvIIx 85763 659038
RpWCElzfAx 85763 635607
IkaKJZPieJ 85763 559902
mkyXWDTpNl 85763 519387
wjzsngfzSB 85763 495633
bsCtmvnnIg 85763 480729
dJdwxDtUdv 85763 432417
nczIxwHfRY 85763 389753
dwpfcRKhib 85763 383872
tRaqxMwDci 85763 253304
LWFWFpBBha 85763 191772
xtmOyAtSgJ 85763 157188
qEWTkvQGtY 85763 141334
ebbOYPzDOY 85763 94358
jZSXsdSUXR 85763 1308
